modules/virt-measuring-latency-vm-secondary-network.adoc

```diff
@@ -215,14 +222,14 @@ $ oc apply -f <latency_job>.yaml
 +
 [source,terminal]
 ----
-$ oc wait --for=condition=complete --timeout=10m job.batch/kubevirt-vm-latency-checkup -n kiagnose
+$ oc wait job kubevirt-vm-latency-checkup -n kiagnose --for condition=complete --timeout 6m
 ----
 
-. Review the results of the latency checkup by retrieving the status of the `ConfigMap` object. If the measured latency is greater than the value of the `spec.param.max_desired_latency_milliseconds` attribute, the checkup fails and returns an error.
+. Review the results of the latency checkup by retrieving the status of the `ConfigMap` object. If the maximum measured latency is greater than the value of the `spec.param.max_desired_latency_milliseconds` attribute, the checkup fails and returns an error.
```

Comment:
I know the docs aren't super consistent with this, but the contrib guide still includes this guidance so I feel compelled to mention it:
```
If a step in a procedure is to run a command, make sure that the step text includes an explicit instruction to "run" or "enter" the command. In most cases, use one of the following patterns to introduce the code block:

<Step description> by running the following command:

<Step description> by entering the following command:

<Step description>, run the following command:

<Step description>, enter the following command:
```